src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -473,15 +471,14 @@ public class SimpleEvents {
 				.examples("on slime split:")
 				.since("2.2-dev26");
 		Skript.registerEvent("Resurrect Attempt", SimpleEvent.class, EntityResurrectEvent.class, "[entity] resurrect[ion] [attempt]")
```

Comment:
Could you maybe set EntityRessurectEvent's default listening behavior to `ListeningBehavior.ANY`? Because I think it's more intuitive this way

Reply:
I don't think you have to change any of the current API. Couldn't you just make it a separate class extending `SkriptEvent` and modify the `EventData` in its init? I don't see why that wouldn't work

Reply:
I had to make it a little more complicated to make sure players can still choose a specific behavior without it being overridden by the default behavior, but it does fix the breaking change.